extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -349,6 +364,11 @@ export class SystemLayer {
     this.storeService_.subscribe(StateProperty.PAGE_HAS_AUDIO_STATE, audio => {
       this.onPageAudioStateUpdate_(audio);
     }, true /** callToInitialize */);
+
+    this.storeService_.subscribe(StateProperty.STORY_HAS_SIDEBAR_STATE,
+        hasSidebar => {
+          this.checkSidebarDisplay_(hasSidebar);
```

Comment:
Nit: mostly for consistency, it'd be great to keep this event handler more global, and maybe name it `onHasSidebarStateUpdate` like the other ones

Reply:
Sorry I've been trying to look around the code, but in this context can you point me to an example of a more global event handler?